content/sensu-go/5.15/sensuctl/reference.md

```diff
@@ -798,14 +802,71 @@ fetching bonsai asset: sensu/sensu-slack-handler:1.0.3
 added asset: sensu/sensu-slack-handler:1.0.3
 {{< /highlight >}}
 
-To check your Sensu backend for assets which have newer versions available on Bonsai, use `sensuctl asset outdated`. This will print a list of assets installed in the backend whose version is older than the newest version available on Bonsai.
+### Check your Sensu backend for outdated assets
+
+To check your Sensu backend for assets that have newer versions available on Bonsai, use `sensuctl asset outdated`. This will print a list of assets installed in the backend whose version is older than the newest version available on Bonsai.
 {{< highlight shell >}}
 sensuctl asset outdated
           Asset Name                  Bonsai Asset          Current Version  Latest Version
 ----------------------------  ----------------------------  ---------------  --------------
 sensu/sensu-influxdb-handler  sensu/sensu-influxdb-handler       3.1.1            3.1.2
 {{< /highlight >}}
 
+### Extend sensuctl with commands
+
+Use `sensuctl command` to install, execute, list, and delete commands from Bonsai or a URL.
+
+#### Install commands
+
+To install a sensuctl command from Bonsai or a URL, use:
+{{< highlight shell >}}
+sensuctl command install [ALIAS] ([NAME]:[VERSION] | --url [ARCHIVE_URL] --checksum [ARCHIVE_CHECKSUM]) [flags]
+{{< /highlight >}}
+
+Replace `[ALIAS]` with a unique name for the command. `[ALIAS]` is required.
+
+You must specify **either** `[NAME]` (with optional `[:VERSION]`) **or** `--url ARCHIVE_URL --checksum ARCHIVE_CHECKSUM`.
+
+Replace `[NAME]` with the name of the asset from Bonsai.
+`[:VERSION]` is only required if you require a specific version or are pinning to a specific version. If you do not specify a version, sensuctl will fetch the latest version from Bonsai.
+
+Replace `[flags]` with the resource-specific flags you want to use.
+
+For example, to install a command from the [Sensu Go Email Handler Plugin][35] with the [`-t` and -`f` flags][36] to retrieve the "To" and "From" email addresses:
+
+{{< highlight shell >}}
+sensuctl command install [ALIAS] (sensu/sensu-email-handler:0.2.0) -tf
+...
+**NEEDED**
+...
+{{< /highlight >}}
+
+To install a command from a URL:
+
+{{< highlight shell >}}
+sensuctl command install [ALIAS] (--url https://gist.githubusercontent.com/amdprophet/43790e007e949aed19131b100e1576af/raw/0c3a1af85481db2bcfbf9f7be513fa7fc0d0fc18/asset.yml --checksum **NEEDED*)
+...
+**NEEDED**
+...
+{{< /highlight >}}
+
+#### Execute commands
+
+Use `sensuctl command exec [ALIAS] [args] [flags]` to execute a sensuctl command plugin via its asset's bin/entrypoint executable.
+
+Replace `[ALIAS]` with with a unique name for the command. `[ALIAS]` is required.
+
+Replace `[args]` with **NEEDED**. `[args]` is passed through to the bin/entrypoint executable.
```

Comment:
This is correct however we should mention that in order for flags to be passed to the `bin/entrypoint` executable, they must be specified after a double dash surrounded by spaces ` -- `.

For example:
```
sensuctl command exec mycommand arg1 arg2 --cache-dir /tmp -- --flag1 --flag2=value
```

The `--cache-dir` flag will be parsed by sensuctl, but all flags after the `--` will be parsed by `bin/entrypoint`. The full command being run by sensuctl exec in this case would be:

```
bin/entrypoint arg1 arg2 --flag1 --flag2=value
```